packages/graphback/src/schema/schemaTemplate.ts

```diff
@@ -30,53 +30,107 @@ const filters = (defs: TargetType[]): string => {
 
 const imports = `import gql from 'graphql-tag'`
 
+/**
+ * Patch together generated and custom queries
+ * @param queries queries generated from the types (CRUD)
+ * @param custom custom queries inputted by the user
+ */
+const generateQueries = (queries: string[], custom: Field[]) => {
+  if(custom.length) {
```

Comment:
This code can be much simpler. We can calculate this map beforehand and have:
```
${queries.join('\n')}
${customQueries.join('\n')}
```

Reply:
Yeah but since those are string templates, there will be unrequired whitespaces in the output in case of no custom input